bosh_cli/lib/cli/commands/job_management.rb

```diff
@@ -104,6 +110,10 @@ def parse_manifest(operation)
           err('Cannot handle both --hard and --soft options, please choose one')
         end
 
+        if !fix_option_allowed?(operation) && fix?
+          err("--fix option only makes sense for 'recreate' operation")
```

Comment:
If we control which commands have the `--fix` option, do we need to be performing this check as well - I thought the regular opts/args parser would error?